### ISSUE TITLE
 An active access token must be used to query information about the current user.

### ISSUE THREAD

**Issue by thinnv** [abjerner/Skybrud.Social.Facebook#19]
I use Skybrud.Social  v0.9.5.0: 
 Exception:Skybrud.Social.Facebook.Exceptions.FacebookException: An active access token must be used to query information about the current user.
   at Skybrud.Social.Facebook.Responses.FacebookResponse.ValidateResponse(SocialHttpResponse response, JsonObject obj)
   at Skybrud.Social.Facebook.Responses.Users.FacebookUserResponse.ParseResponse(SocialHttpResponse response)
   at MVCForum.Website.Controllers.OAuthControllers.FacebookOAuthController.FacebookLogin()

**Comment by abjerner** [comment]
@thinnv Do you have some more information about the error? I can't really tell much by the exception message and stack trace, except that your access tokens seems to be invalid.

Also, `v0.9.5` isn't the newest version, so try and update to `v0.9.6` to see whether the error persists ;)

**Comment by thinnv** [comment]
I use v0.9.6 and Api v2.9:
Error FacebookOAuthClient.GetAccessTokenFromAuthCode returns null
but i use  v0.9.6 and Api v2.8 --> return OK

**Comment by abjerner** [comment]
@thinnv I just did a few tests with `v0.9.6` and `v2.9`, and they all worked fine. It might an edge in your end causing an error, so please provide as much information as you can.

Also, `GetAccessTokenFromAuthCode` returning `null` was the issue fixed in `v0.9.6`, so please confirm that you are in fact using `v0.9.6`. If you have multiple projects in your Visual Studio solution, you'll most likely need to update Skybrud.Social in each project.

You can try writing out these in a Razor view to be sure:

```
<pre>API version: @client.Version</pre>
<pre>Skybrud.Social: @SocialUtils.GetVersion()</pre>
```

where `client` is your instance of `FacebookOAuthClient`.

I hope that helps ;)
